README.md

```diff
@@ -45,7 +46,23 @@ source ~/rds/rds-ukaea-mast-sPGbyCAPsJI/uda-ssl.sh
 
 You should now be able to run the following commands.
 
-## Local Ingestion
+### Submitting runs on CSD3
+
+1. First submit a job to collect all the metadata:
+
+```sh
+sbatch ./jobs/metadata.csd3.slurm.sh
+```
+
+2. Then submit an ingestion job
+
+```sh
+sbatch ./jobs/ingest.csd3.slurm.sh campaign_shots/tiny_campaign.csv s3://mast/test/shots/ amc
```

Comment:
User will need the .s5cfg file, not sure if that should be specified in the readme?